### PR TITLE
whos that pokemon

### PR DESCRIPTION
**Pull request by jadxie**
here goes...

### CODE REVIEW

File: bot/whos_that_pokemon_manager.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            sprite = pokemon[sprites][front_default]
+            self.correct_answer = pokemon['name']
+            self.awaiting_answer = True
+            return sprite
+    
+    def check_response(self, user_id, msg):
+        if self.awaiting_answer and re.search('it\'?s', msg):
+            if re.search(self.cached_correct_response, msg):
+                response = reveal_correct_answer + " You go " + user_id + "!"
+                return response
```
Comment (jadxie) [inline bot/whos_that_pokemon_manager.py]: missed the else for this. knew I forgot something

**Comment by jadxie** [comment]
(I see several issues already that I'm working on, just needed it all to be laid out for myself before crashing zac)

File: resources/pokemon_incorrect.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ You aren't very good at this, are you.
+ I'm very disappointed in you.
+ I expected better from you.
+ This isn't Digimon...
+ Ten points from Hufflepuff.
+ Ten points from Ravenclaw.
+ That was quite possibly the worst guess ever.
+ _Le sigh._
+ So wrong. So very, very wrong.
+ You might consider investing in a Pokedex...
```
Comment (keenns) [inline resources/pokemon_incorrect.txt]: I am lovin' these responses lol

File: resources/pokemon_correct.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Bless my Shellder you got it right!
+Dayum you good.
+Nailed it :+1:
+Ten points to Hufflepuff.
+Ten points to Ravenclaw.
+You _are_ the very best!
+:tada:
+I salute you!
+_Le YES!_
+You are a Pokemon _Master!_
```
Comment (keenns) [inline resources/pokemon_correct.txt:11]: I love how Gryffindor and Slytherin can't get points :ok_hand:

**Comment by keenns** [comment]
It's lookin' super sweet to me! my newbie python skills have no helpful comments to contribute it's lookin great :+1:

**Comment by jadxie** [comment]
Lol my brain was mushy and I just wanted to see the diff. (Stupid command line ugliness.) Thanks for the comments! Still some logic to hash out and then time to crash zac a few dozen times. Fun for tomorrro! JOIN THE PYTHON PARTY!!

**Comment by keenns** [comment]
Also if seeing the difference with yer changes is something very useful to ya (which it is to me) then i highly recommend you use vs code! It has a whole featurethat easily lets you compare your changes, but you can see the entire file nicely rather than just where you made changes unlike github provides. Also vs code is just wonderful in general. #sponsor #vscode4thewin

**Comment by jadxie** [comment]
Sweet, checking it out!

**Pull request merged**